.circleci/config.yml

```diff
@@ -0,0 +1,74 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: ruby:2.5.0-alpine
+        environment:
+          BUNDLE_RETRY: 3
+          BUNDLE_PATH: vendor/bundle
+    working_directory: ~/rds-scheduler/src
+    steps:
+      - checkout:
+          path: ~/rds-scheduler
+      - run:
+          name: Install build dependencies
+          command: apk add -U g++ make
+      - run:
+          name: Install ruby dependencies
+          command: bundle install --gemfile=Gemfile --path=vendor/bundle
+      - run:
+          name: Run code analyser
+          command: bundle exec rubocop
+      - run:
+          name: Run RSpec
+          command: bundle exec rspec spec/
+  build:
+    machine: true
+    steps:
+      - checkout
+      - run:
+          name: Build Image
+          command: docker build -t appvia/rds-scheduler:$CIRCLE_SHA1 .
+  push_latest:
+    machine: true
+    steps:
+      - checkout
+      - run:
+          name: Push to Quay with tag latest
+          command: |
+            docker build -t quay.io/appvia/rds-scheduler:latest .
+            docker login quay.io --username $QUAY_USERNAME --password $QUAY_PASSWORD
+            docker push quay.io/appvia/rds-scheduler:latest
+  push_tag:
```

Comment:
Should this step (and the one above) make use of [filters](https://circleci.com/docs/2.0/configuration-reference/#filters-1) to ensure they only get run on specific occasions? In this case, when a Git tag is pushed. And in the case above, when `master` is pushed.

Reply:
Oh sorry, I see it now below in the `workflows` section :)